scripts/create-kubestellar-demo-env.sh

```diff
@@ -119,26 +133,18 @@ checking_cluster() {
     done
 }
 
-# Function to check if a command exists
-command_exists() {
-    command -v "$1" >/dev/null 2>&1
-}
-##########################################
-
 echo -e "\nStarting environment clean up..."
 echo -e "Starting cluster clean up..."
 
 if command_exists "k3d"; then
     cluster_clean_up "k3d cluster delete kubeflex" &
     cluster_clean_up "k3d cluster delete cluster1" &
     cluster_clean_up "k3d cluster delete cluster2" &
-    wait
```

Comment:
We deliberately put a wait between the k3d cleanup and the kind cleanup because of observed problems with doing them concurrently.